students/sendecki_andrzej/lesson_05_lists/lesson7-swap_min_and_max.py

```diff
@@ -0,0 +1,36 @@
+#Given a list of unique numbers, swap the minimal and maximal elements of this list. Print the resulting list. 
+
+def find_min(my_list):
+    min_num = my_list[0]
+
+    for i in range(len(my_list)):
+        if my_list[i] < min_num:
+            min_num = my_list[i]
+
+    return min_num
+
+def find_max(my_list):
+    max_num = my_list[0]
+
+    for i in range(len(my_list)):
+        if my_list[i] > max_num:
+            max_num = my_list[i]
+
+    return max_num
+
+
+numbers = [1, 2, 6, 4, 5, 8, 0, -9, 3]
+
+print("Original list")
+print(numbers)
+
+min_index = numbers.index(find_min(numbers))
```

Comment:
There are already built-in functions like ` min` and `max`, don't need to write it yourself or even import it :D

Reply:
Check out how full solution could look like:
```
a = [int(i) for i in input().split()]

min_idx = a.index(min(a))
max_idx = a.index(max(a))

a[min_idx], a[max_idx] = a[max_idx], a[min_idx]
print(' '.join([str(i) for i in a]))
```